ai/STA/Strategy/prepare_kickoff_defense.py

```diff
@@ -49,12 +47,13 @@ def __init__(self, p_game_state):
                              Role.SECOND_DEFENCE: defense_bottom_position}
 
         for role, position in role_to_positions.items():
-                player = self.assigned_roles[role]
-                node_go_to_position = self.create_node(role, GoToPositionPathfinder(self.game_state, player, position))
-                node_stop = self.create_node(role, Stop(self.game_state, player))
-                player_arrived_to_position = partial(self.arrived_to_position, player)
+            position.orientation = np.pi
```

Comment:
should be a pose, position dont have orientation

Reply:
I think @SimLemay  fix this in his pull request

Reply:
role_to_positions already contains Poses. I can rename it to role_to_poses for consistency